examples/basic/app.R

```diff
@@ -5,7 +5,7 @@ library(shiny.router)
 menu <- (
   tags$ul(
     tags$li(a(class = "item", href = "/", "Page")),
-    tags$li(a(class = "item", href = "/other", "Other page"))
+    tags$li(a(class = "item", href = route_link("other"), "Other page"))
```

Comment:
Please export `route_link` function or use `shiny.router:::route_link` instead.

Reply:
I fixed semantic example now.

Reply:
and NAMESPACE updated